acct.bionimbus.org/etlMapping.yaml

```diff
@@ -1,5 +1,5 @@
 mappings:
-  - name: acct_etl
+  - name: acct.bionimbus.org_subject
```

Comment:
Why did this row change?

Reply:
This is working by design.
Our automation names the Elastic Search index according to the web address / hostname of the target environment.
This should be harmless.